lib/replicate.js

```diff
@@ -97,6 +97,11 @@ Replication.prototype.cancel = function () {
 
 Replication.prototype.ready = function (src, target) {
   var self = this;
+  if (self._readyCalled) {
+    return;
+  }
+  self._readyCalled = true;
+
```

Comment:
@dholth This is what I was talking about. ^ Maybe it's 6 of one, half a dozen of the other, but I find this more readable than making `ready()` a noop.